style2.css

```diff
@@ -121,58 +120,55 @@ img {
   float: left;
 }
 
-.contact-type{
-  display:none;
+.contact-type {
+  display: none;
 }
 
-
-a[href='mailto:jad@codi.tech']{
-background: url('images/email.png') no-repeat 6px center;
-background-size: 85px 85px;
-width: 85px;
-height: 85px;
-display: block;
-}
-a[href='tel:+96170261482']{
-background: url('images/phone.png') no-repeat 6px center;
-background-size: 85px 85px;
-width: 85px;
-height: 85px;
-display: block;
+a[href='mailto:jad@codi.tech'] {
```

Comment:
Instead of that, why not something like:

```css
.contact-email a{}
.contact-skype a{}
...
```

Content of those tags might change!